### DISCUSSION TITLE
I've prepared MR "[Modesecurity] Upgrade modesecurity ... #36" how could I get a snapshot artifact to test ?

### DISCUSSION THREAD

**Discussion by yaandy** [bytedeco/javacpp-presets#1491] [General]
I try to run action on a fork but don't understand how to get a snapshot after successful build, maven oss.sonatype.org Snapshots does not contain an artifact, action also does not contain any artifacts. Maven report that successfully install it into local repo in the logs.

**Comment by saudet** [comment]
If you have an account for Sonatype OSSRH, you can set those secrets in your own fork:
https://github.com/bytedeco/javacpp-presets/blob/master/.github/workflows/modsecurity.yml#L15-L18
